### DISCUSSION TITLE
Remove the default "Create XXX" from options

### DISCUSSION THREAD

**Discussion by assaf-itzikson** [JedWatson/react-select#4612] [Q&A] [answered]
Is there a way to remove the default option to create something in a CreatableSelect dropdown?
I'm talking about the `Create "166"` on the bottom of the list. I don't need it because everything I want to add to the list must have the `@` symbol on it.
![image](https://user-images.githubusercontent.com/75263691/119995622-af925f00-bfc5-11eb-80a8-65dcadf34f05.png)
Thank you.

**Comment by Rall3n** [comment, marked as answer]
Hello @assaf-itzikson,

do I understand you correctly, that you don't want the `Create ...` option? If so, I recommend not using the `Creatable` component and instead going for the default `Select` component, as the option is crucial for the functionality of the `Creatable` component and removing it would just leave unused code in your bundle.

If you just want the option to appear for certain conditions (e.g. value starting with `@`), I recommend using the `isValidNewOption` prop, which accepts a function as its value. This function gets the value of the search input as its first argument.

```jsx
<Creatable isValidNewOption={(inputValue) => inputValue && inputValue.startsWith('@') && /* Check for existing value */} />
```

**Reply by assaf-itzikson** [reply]
Thank you, I will try that and update.

**Reply by assaf-itzikson** [reply]
> Hello @assaf-itzikson,
> 
> do I understand you correctly, that you don't want the `Create ...` option? If so, I recommend not using the `Creatable` component and instead going for the default `Select` component, as the option is crucial for the functionality of the `Creatable` component and removing it would just leave unused code in your bundle.
> 
> If you just want the option to appear for certain conditions (e.g. value starting with `@`), I recommend using the `isValidNewOption` prop, which accepts a function as its value. This function gets the value of the search input as its first argument.
> 
> ```js
> <Creatable isValidNewOption={(inputValue) => inputValue && inputValue.startsWith('@') && /* Check for existing value */} />
> ```

sorry for the late reply, I just had the opportunity to talk to my dev and he confirmed that the method you suggested helped with removing the `Create...` option.
![Screenshot 2021-06-22 at 13 55 01](https://user-images.githubusercontent.com/75263691/122928225-903ae780-d361-11eb-82fd-cb5de45d2f1c.png)
so, thank you once again.
